Services/CO.CDP.Tenant.WebApi/appsettings.AwsDevelopment.json

```diff
@@ -1,4 +1,25 @@
 {
+  "Serilog": {
+    "Using": [ "Serilog.Sinks.AwsCloudWatch" ],
+    "MinimumLevel": {
+      "Default": "Information",
+      "Override": {
+        "Microsoft.AspNetCore.Mvc": "Warning",
+        "Microsoft.AspNetCore.Routing": "Warning",
+        "Microsoft.AspNetCore.Hosting": "Warning"
+      }
+    },
+    "WriteTo": [
+      {
+        "Name": "AmazonCloudWatch",
+        "Args": {
+          "logGroup": "your-app",
```

Comment:
Is it `your-app` 'cause it's going to be overridden? Otherwise, can we give it a better name?